storage/src/store/operation.rs

```diff
@@ -74,7 +74,7 @@ pub enum Variable<K: Array, V: Codec> {
     // Operations for mutable stores.
     Delete(K),
     Update(K, V),
-    CommitFloor(u64),
+    CommitFloor(UInt<u64>),
```

Comment:
You should be able to keep this as `u64` and just use `UInt` during `Write`/`Read`.

Reply:
Example: https://github.com/commonwarexyz/monorepo/blob/fd795b569f57832a029e0ba86370f9744c25c88c/examples/vrf/src/handlers/wire.rs#L14-L46

Reply:
done!